packages/amplify-go-function-runtime-provider/src/runtime.ts

```diff
@@ -63,6 +70,9 @@ const isBuildStale = (resourceDir: string, lastBuildTimeStamp: Date, outDir: str
   return !!fileUpdatedAfterLastBuild;
 };
 
+/**
+ *
+ */
```

Comment:
Sorry to be nitpicky because I know these were auto generated, but can you remove these empty function definitions? You should be able to make these changes without running the lint filter with the `--no-verify` flag.

Reply:
Changes are done.